src/assets/mock-data/mockPassengers.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "name": "Martha Marie Smith",
+        "name": "Janice Smith",
```

Comment:
What did "Martha Marie" do to you ? 😁

Reply:
She didn't do anything, she just wasn't needed for the final design.